common/kvstore/src/main/java/org/apache/spark/util/kvstore/RocksDBIterator.java

```diff
@@ -36,20 +41,24 @@ class RocksDBIterator<T> implements KVStoreIterator<T> {
   private final byte[] indexKeyPrefix;
   private final byte[] end;
   private final long max;
+  private final Cleaner.Cleanable cleanable;
 
   private boolean checkedNext;
   private byte[] next;
   private boolean closed;
   private long count;
 
-  RocksDBIterator(Class<T> type, RocksDB db, KVStoreView<T> params) throws Exception {
+  RocksDBIterator(Class<T> type, RocksDB db, KVStoreView<T> params,
+      ConcurrentLinkedQueue<Reference<RocksDBIterator<?>>> iteratorTracker,
```

Comment:
I'm more inclined to add two helper methods in RocksDB to get `iteratorTracker` and `_db`, but I'm also open to other suggestions.

Reply:
thanks, done